Buildings/Experimental/DHC/Networks/Connections/Examples/Connection1PipeExample.mo

```diff
@@ -0,0 +1,150 @@
+    lPip=lDis,
+    rPip=0.04,
+    thiGroLay=1.1,
+    nSta=5,
+    nSeg=1)
+    annotation (Placement(transformation(extent={{-11,-10},{11,10}},
+        rotation=0,
+        origin={-11,-30})));
+
+  Modelica.Blocks.Sources.Step mCon_flow_nominal_exp(height=mCon_flow_nominal,
+      startTime=1800)
+    annotation (Placement(transformation(extent={{-80,70},{-60,90}})));
+equation
+  connect(pumDisAutoSize.port_b, connection1PipeAutosize.port_aDis) annotation (
+     Line(points={{22,16},{10,16},{10,50},{20,50}}, color={0,127,255}));
+  connect(bouDisAutoSize.ports[1], connection1PipeAutosize.port_aDis)
+    annotation (Line(points={{0,10},{10,10},{10,50},{20,50}}, color={0,127,255}));
+  connect(pumDisAutoSize.port_a, connection1PipeAutosize.port_bDis) annotation (
+     Line(points={{42,16},{50,16},{50,50},{40,50}}, color={0,127,255}));
+  connect(mDis_flow_nominal_exp.y, pumDisAutoSize.m_flow_in)
+    annotation (Line(points={{-59,30},{32,30},{32,28}}, color={0,0,127}));
+  connect(connection1PipeAutosize.port_bCon, pumConAutoSize.port_a) annotation (
+     Line(points={{30,60},{30,62},{16,62},{16,80},{20,80}}, color={0,127,255}));
+  connect(pumConAutoSize.port_b, connection1PipeAutosize.port_aCon) annotation (
+     Line(points={{40,80},{42,80},{42,62},{36,62},{36,60}}, color={0,127,255}));
+  connect(pumDisPlugFlow.port_b, connection1PipeplugFlow.port_aDis) annotation (
+     Line(points={{22,-80},{12,-80},{12,-50},{20,-50}}, color={0,127,255}));
+  connect(bouDisPlugFlow.ports[1], connection1PipeplugFlow.port_aDis)
+    annotation (Line(points={{0,-80},{12,-80},{12,-50},{20,-50}},   color={0,127,
+          255}));
+  connect(pumDisPlugFlow.port_a, connection1PipeplugFlow.port_bDis) annotation (
+     Line(points={{42,-80},{50,-80},{50,-50},{40,-50}}, color={0,127,255}));
+  connect(mDis_flow_nominal_exp.y, pumDisPlugFlow.m_flow_in) annotation (Line(
+        points={{-59,30},{-52,30},{-52,-64},{32,-64},{32,-68}}, color={0,0,127}));
+  connect(connection1PipeplugFlow.port_bCon, pumConPlugFlow.port_a) annotation (
+     Line(points={{30,-40},{30,-38},{18,-38},{18,-20},{20,-20}}, color={0,127,255}));
+  connect(pumConPlugFlow.port_b, connection1PipeplugFlow.port_aCon) annotation (
+     Line(points={{40,-20},{44,-20},{44,-38},{36,-38},{36,-40}}, color={0,127,255}));
+  connect(pipeGroundCoupling.heatPorts[1], connection1PipeplugFlow.heatPortDis)
+    annotation (Line(points={{-11,-35},{-11,-47.4},{25,-47.4}},
+        color={127,0,0}));
+  connect(mCon_flow_nominal_exp.y, pumConAutoSize.m_flow_in) annotation (Line(
+        points={{-59,80},{0,80},{0,98},{30,98},{30,92}}, color={0,0,127}));
+  connect(pumConPlugFlow.m_flow_in, pumConAutoSize.m_flow_in) annotation (Line(
+        points={{30,-8},{30,-4},{60,-4},{60,98},{30,98},{30,92}}, color={0,0,127}));
+  annotation (
+    Icon(
+      coordinateSystem(
+        preserveAspectRatio=false)),
+    Diagram(
+        coordinateSystem(
+        preserveAspectRatio=false)),
```

Comment:
I don't think these annotations are needed.

Reply:
You are right those annotation are not necessary. I also removed them from `Buildings/Experimental/DHC/Networks/Connections/Examples/Connection2PipeExample.mo`.